src/main/java/backend/Teamoverview/ContactWithHeadquartersRepository.kt

```diff
@@ -0,0 +1,19 @@
+package backend.teamoverview
+
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.CrudRepository
+import org.springframework.data.repository.query.Param
+
+interface ContactWithHeadquartersRepository : CrudRepository<ContactWithHeadquarters, Long> {
+
+    fun findAllByTeamId(teamId: Long): Iterable<ContactWithHeadquarters>
+
+    @Query("""
+        select *
```

Comment:
can't we do this using the normal query language?

Reply:
limit is not available jpql 😭

Reply:
But if you can find a better way to run this query without limit, by all means